src/TableFooter.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+const TableFooter = ({dataArrayLength, currentPage, entriesPerPage, onChange, dataLength}) => {
+  const pageNumbers = []
+  for (let i = 1; i <= Math.ceil(dataArrayLength / entriesPerPage); i++) {
+    pageNumbers.push( i === currentPage ? <li className={`current`} key={`bottom${i}`}>{currentPage}</li> :
+      <li key={`bottom${i}`}><a onClick={() => onChange(i)}>{i}</a></li>)
+  }
+
+  return (
+    <div className="row expanded padding-top-medium">
+      <div className={`small-6 columns`}>
+        {
+          dataArrayLength === 0 ?
+            `There are no results for this query.` :
+            `Showing ${dataArrayLength} out of ${dataLength} experiments.`
```

Comment:
[The suggestion from the EBI framework](https://ebiwd.github.io/EBI-Pattern-library/sample-site/boilerplate/search-results.html#) is to display the text *Showing X results from a total of Y (page M of N).*. Besides that we should additionally show a different message when:
1. There are no experiments at all (this should never happen, but hey, who knows?)
2. No experiments are shown because a query doesn’t match

Some suggestions, feel free to reword them as you see fit (1a is the easter egg funny proposal, 1b is the boring science option):
1. a. Nothing to see here. Move along!
    b. No experiments have been loaded
2. No experiments could be found for this query

Also, if there’s also one page of results, I think the *(page M of N)* can be omitted.